feature.add-album-to-queue.md

```diff
@@ -0,0 +1,48 @@
+# Audio Player: Add Album to Queue
+**Authors:** [@szainmehdi](https://github.com/szainmehdi) [@shea786](https://github.com/shea786)  
+**Type:** Feature
+
+## Overview
+Allow the user to add the entire album to the audio player queue
+
+## Why are we making this change?
+So that it is easier for the user, instead of them going to each track to add to queue, one button would add the entire track
+
+## Requirements
+- User should be able to an entire album to the playlist
+- The order of which the tracks are being displayed is the order of which the tracks should be added to the queue
+- User should be able to add all tracks from the `Reciter Profile` page against each table to the queue
+- User should be able to add all the tracks from the `Tracks Show` page to the queue
+- When adding all tracks from an album to the queue, if the queue is shuffled then add the new tracks to the shuffled queue in random order
```

Comment:
I don't like this idea.

In fact, I have something else in mind. 

I think the right approach here is:

- A `play` button is shown on albums.
- A `play` button is shown on the "More From This Album" card.
- Clicking the `play` button discards the current queue and shuffle state, and plays the selected album from top to bottom.

Reply:
I don't think we should be doing that,

In my view, if we have gone through and added tracks to our queue, and later we click play against an album, our current queue will change to the album tracks,
I personally would get really annoyed after it took me so long to add tracks to my queue for it to get overwritten. 

If users don't want the current tracks from the current queue, they can always go and remove them manually right?

Reply:
Maybe as per your mockups, we should have two buttons,

Play which would overwrite the queue
And add to queue which would add as per my logic above

That would be better I think